### ISSUE TITLE
Cannot load a character

### ISSUE THREAD

**Issue by Toeknives** [dalaranwow/dalaran-wow#4301]
Hi,

I recently created an account and character.  Great fun to be back!  Yesterday I entered a dungeon LFG and disconnected upon entering the group.  After logging back in I left the dungeon and now cannot log back in to this character (Toeknives).  When I try to login the loading screen stops at 50% and I get the following error message.  I created a new character and all is fine.
 
ERROR #134 (0x85100086) Fatal Condition
Program:	C:\Users\Andy\My Downloads\World of Warcraft Client 3.3.5a - Ready to Play\Wow.exe

Please help my stranded paladin!\

Best,
Toeknives

[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
The issuetracker is for serverside and gamemechanic issues only. Please write an Ingame-issue to a Gamemaster and the Gamemaster will then help you as soon as he can. Thank you.

![GitHub Logo](http://i.imgur.com/bYkYeGQ.png)
